src/server/seeds/dev/005_00_ports.js

```diff
@@ -0,0 +1,22 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("ports")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("ports").insert([
+        {
+          id: 1,
+          name: "port1"
+        },
+        {
+          id: 2,
+          name: "port2"
+        },
+        {
+          id: 3,
+          name: "port3"
+        }
+      ]);
+    });
+};
```

Comment:
Please give some more realistic names to the ports and add some more.

Can you add the ports from this list and link a waypoint to each of them with the coordinates listed in the json file?

https://trello-attachments.s3.amazonaws.com/5d741a742ea45775126dc294/5d74f07f20624e860839e985/74258c233230498015ce407a2a4ff688/harbour_info.json